packages/gatsby/src/utils/babel-loader.js

```diff
@@ -24,13 +25,24 @@ const {
 module.exports = babelLoader.custom(babel => {
   const toReturn = {
     // Passed the loader options.
-    customOptions({ stage = `test`, reactRuntime = `classic`, ...options }) {
+    customOptions({
+      stage = `test`,
+      reactRuntime = `classic`,
+      rootDir = process.cwd(),
+      ...options
+    }) {
       return {
         custom: {
           stage,
           reactRuntime,
         },
         loader: {
+          cacheIdentifier: JSON.stringify({
+            browerslist: getBrowsersList(rootDir),
+            babel: babel.version,
```

Comment:
Wouldn't Babel assert the version that created the cache matches its current version?

Reply:
no :) This is their default https://github.com/babel/babel-loader/blob/fd7d3d488d245fa673aafc513ca839252509c3fe/src/index.js#L184-L193 theoretically it's version agnostic.